apps/src/code-studio/DisabledBubblesAlert.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react';
+import Alert from '@cdo/apps/templates/alert';
+import i18n from '@cdo/locale';
+
+const styles = {
+  bold: {
+    fontFamily: '"Gotham 5r", sans-serif'
+  }
+};
+
+/**
+ * Component that displays a small notification at the top of the page when
+ * postMilestone is disabled (and thus progress is disabled)
+ */
+const DisabledBubblesAlert = React.createClass({
+  propTypes: {
+    isHocScript: PropTypes.bool.isRequired
+  },
+
+  getInitialState() {
+    // Once alert has been dismissed, don't show again.
+    const disabledBubblesAlertSeen = sessionStorage.getItem('disabledBubblesAlertSeen');
```

Comment:
Oh, I see, [over here](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API):

> sessionStorage maintains a separate storage area for each given origin

That's not clear to me from the actual `window.sessionStorage` page at all.  Something called a "page session" sounds like it would be specific to a particular page, not global to an origin.